composer.json

```diff
@@ -12,8 +12,9 @@
     }
   ],
   "require": {
-    "php": "~8.1.0",
-    "pimcore/pimcore": "^10.5"
+    "php": "^8.1",
```

Comment:
Ok, this didn't work very well...
![image](https://github.com/teamneusta/pimcore-fixture-bundle/assets/5848798/4161e157-5c73-4489-a691-7941230ca711)

Reply:
And mentioning this article [here](https://medium.com/geekculture/tilde-vs-caret-in-package-json-whats-the-difference-2682b7e6ecf8#:~:text=We've%20seen%20that%20the,keeping%20the%20major%20version%20fixed.), I would think caret operator is better in that way, isn't it?!

Reply:
Well, `^8.1` and `~8.1` behave in the same way, and the former is preferable, so the issue is correct.

But we want `~8.1.0` instead (like it was before) and now with Pimcore 11 suport it should be `~8.1.0 || ~8.2.0 || ~8.3.0`.

Reply:
> And mentioning this article [here](https://medium.com/geekculture/tilde-vs-caret-in-package-json-whats-the-difference-2682b7e6ecf8#:~:text=We've%20seen%20that%20the,keeping%20the%20major%20version%20fixed.), I would think caret operator is better in that way, isn't it?!

No, it is not (for PHP itself), because the caret operator expects SemVer, but PHP does not adhere to SemVer, so the tilde operator should be used.